main.tf

```diff
@@ -1,40 +1,60 @@
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+data "aws_secretsmanager_secret" "db_secret" {
```

Comment:
in general, you could use `name` parameter, instead of `arn`, it will simplify the the code for reading a bit